charts/lfx-v2-committee-service/templates/ingressroute.yaml

```
@@ -0,0 +1,28 @@
+  entryPoints:
+    - web
+    - websecure
+  routes:
+    - kind: Rule
+      match: >-
+        Host(`{{.Values.ingress.hostname}}`) &&
+        (Path(`/committees`) || PathPrefix(`/committees`))
+      priority: 10
+      middlewares:
+        {{- if .Values.heimdall.enabled }}
+        - name: heimdall
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)